admin-site/src/Button.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+export let buttonText = "";
+export let buttonType = 'btn-outline-primary btn-block';
+export let dataTarget = undefined;
+export let value = "";
+export let disabled = undefined;
+export let bsSizePosition = "col-md-12";
+export let iconLeft = undefined;
+export let iconRight = undefined;
+//for modal close
+export let dataDismiss = undefined;
+export let ariaLabel = undefined;
+export let buttonHtml = "";
+
+</script>
+<style>
+  button {
+	  margin-bottom:0.5rem;
+  }
+  span{
+      vertical-align: middle;
+  }
+  .abs-left { 
+    position: absolute;
+    left: 0.5rem;
+  }
+  .backButton{
+      vertical-align: text-bottom;
+  }
+  
+</style>
+
+
+    <div class={bsSizePosition}>
+    
+            <button on:click
+                type="button" 
+                class="btn {buttonType}" 
+                data-toggle="modal" 
+                data-target={dataTarget}
+                data-dismiss = {dataDismiss}
+                aria-label = {ariaLabel}
+                value = {value}
+                disabled = {disabled}>
+
+            <span class={iconLeft}/>
```

Comment:
You may want to check out the Bootstrap Input Group component: https://getbootstrap.com/docs/4.4/components/input-group/

Reply:
Is there a place where we want buttons inline with inputs? So far, the markup inside the button element is only for text inside a button.

Reply:
Ah, got it, this is just i.e. the chevron for the 'back' button.